pkgs/tools/security/wordlists/wfuzz.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, wfuzz
+}:
+
+let
+  shared = "wfuzz";
+
+in stdenv.mkDerivation rec {
+  pname = "wfuzz";
```

Comment:
I don't think this `pname` is correct... Shouldn't it be `wfuzz.lst`?

Reply:
I don't see why: everything is scoped under `wordlists`.
For consistency with the rest, I changed `nmap`'s `pname`.

Reply:
ah ok.

Reply:
It might be appropriate if this would be just a separate output from `wfuzz`, but this depends on the size of the wordlist (I'm not sure what's an appropriate cutoff point; the list seems to have a size of around ~1,2MiB).